content/links/elements-in-angular-demo-github-repository.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Angular Elements: Demo - Using Elements In Another Angular Project.'
+description: 'GitHub Repository for elements-in-angular-demo. Contribute on GitHub.'
+publishedAt: 2020-06-04T13:28:39.082Z
+updatedAt: 2020-06-04T13:28:39.082Z
+url: https://github.com/garygrossgarten/elements-in-angular-demo
+authors:
+  - 'Gary Großgarten'
+tags: 
+  - Angular
+  - Web Components
+---
+![GitHub - garygrossgarten/elements-in-angular-demo: GitHub Repository for elements-in-angular-demo screenshot](assets/img/links/github---garygrossgartenelements-in-angular-demo-github-repository-for-elements-in-angular-demo.png)
+
+This is an addition to my [previous article](https://notiz.dev/blog/create-a-component-library-for-angular-and-the-web) about Angular Elements, where I show how to setup a reusable Angular Component Library with Angular Elements.
```

Comment:
Should it be "where you learn how to ..." instead of "where I show how to..."

Reply:
> Should it be "where you learn how to ..." instead of "where I show how to..."
😹